tests/unit/docker/test_hub_build.py

```diff
@@ -65,6 +65,39 @@ def test_hub_build_push():
     assert manifests[0]['name'] == summary['manifest_info']['name']
 
 
+def test_hub_build_push_push_again():
+    args = set_hub_build_parser().parse_args([os.path.join(cur_dir, 'hub-mwu'), '--push', '--host-info'])
+    summary = HubIO(args).build()
+
+    with open(os.path.join(cur_dir, 'hub-mwu', 'manifest.yml')) as fp:
+        manifest = yaml.load(fp)
+
+    assert summary['is_build_success']
+    assert manifest['version'] == summary['version']
+    assert manifest['description'] == summary['manifest_info']['description']
+    assert manifest['author'] == summary['manifest_info']['author']
+    assert manifest['kind'] == summary['manifest_info']['kind']
+    assert manifest['type'] == summary['manifest_info']['type']
+    assert manifest['vendor'] == summary['manifest_info']['vendor']
+    assert manifest['keywords'] == summary['manifest_info']['keywords']
+
+    args = set_hub_list_parser().parse_args([
+        '--name', summary['manifest_info']['name'],
+        '--keywords', summary['manifest_info']['keywords'][0],
+        '--type', summary['manifest_info']['type']
+    ])
+    response = HubIO(args).list()
+    manifests = response
+
+    assert len(manifests) >= 1
+    assert manifests[0]['name'] == summary['manifest_info']['name']
+
+    # try and push same version again should fail
+    args = set_hub_build_parser().parse_args([os.path.join(cur_dir, 'hub-mwu'), '--push', '--host-info'])
+    summary = HubIO(args).build()
+    print(summary['is_build_success'])
```

Comment:
An assertion for the exception is expected here.

Reply:
Is there a way to restore the PR and edit it? Or should I just create a new one?

Reply:
create a new one better